airbyte-cdk/python/airbyte_cdk/sources/file_based/stream/cursor/default_file_based_cursor.py

```diff
@@ -13,7 +13,7 @@
 
 class DefaultFileBasedCursor(FileBasedCursor):
     DEFAULT_DAYS_TO_SYNC_IF_HISTORY_IS_FULL = 3
-    DATE_TIME_FORMAT = "%Y-%m-%dT%H:%M:%S.%fZ"
+    DATE_TIME_FORMAT = "%Y-%m-%dT%H:%M:%SZ"
```

Comment:
interesting, okay i'll revert my start_date PR back to this then. I was originally converting back out to milliseconds

So do our CAT tests in general just not support anything more granular than seconds? I'm curious because I do see some configs where we use more granular precisions. Or is this specifically to retain the precision being used by S3?

Reply:
got it. thanks for checking!